core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -6330,6 +6202,104 @@ public function initPicker(PickerInterface $picker)
 
 		return $attributes;
 	}
+
+	/**
+	 * @param array  $row   The data record
+	 * @param string $table The table of the data record
+	 */
+	private function getLabel($row, $table = null): string
```

Comment:
Can we rename this method? So it answers the question to "Get label of what?"

Reply:
May be `getRecordLabel`? And then add _Generates the label for a given record as defined in its DCA_ to the docblock.

Reply:
I did not check the function line by line, but I just wanted to note this message overlaps with the universal picker and the improved undo module (#3498), which both need to display the label, too. Is there any way to actually share this logic?

Reply:
@bezin good point. May be a DCA utility service of some sort, that can handle such tasks?